test/DebugInfo/LoadableByAddress-argument.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swift-frontend %s -emit-ir -g -o - | %FileCheck %s
+
+public struct Large {
+  let field1 : Int64 = 1
+  let field2 : Int64 = 2
+  let field3 : Int64 = 3
+  let field4 : Int64 = 4
+  let field5 : Int64 = 5
+  let field6 : Int64 = 6
+  let field7 : Int64 = 7
+  let field8 : Int64 = 8
+}
+
+// CHECK: !DILocalVariable(name: "largeArg", arg: 1
```

Comment:
As this is caused by `LoadableByAddress`, can we also dump the relevant SIL output after the transformations? (not the whole function).

Reply:
That would be nice, but there is an outstanding bug (rdar://problem/35800325) that we can't dump the output of SIL transformations that happen in IRGen.

Reply:
OK, cool, thanks!